lib/logstash/outputs/syslog.rb

```diff
@@ -138,11 +152,11 @@ def receive(event)
 
     if @is_rfc3164
       timestamp = event.sprintf("%{+MMM dd HH:mm:ss}")
-      syslog_msg = "<#{priority.to_s}>#{timestamp} #{sourcehost} #{appname}[#{procid}]: #{event.sprintf(@message)}"
+      syslog_msg = "<#{priority.to_s}>#{timestamp} #{sourcehost} #{appname}[#{procid}]: #{message}"
```

Comment:
With this pull request, every event is passed to a codec. If no explicit `codec` is defined, the config value from `message` is used as `format` for the `plain` codec (https://github.com/logstash-plugins/logstash-output-syslog/pull/17/files#diff-3ccf50b4bfe2babe74a94ff1b3a791a6R123). This ensures backward compatibility, because the `plain` codec does the `sprintf` (https://github.com/logstash-plugins/logstash-codec-plain/blob/master/lib/logstash/codecs/plain.rb#L41).

Unfortunately in #12 there was no spec-test added for the sprintf, which would show now, that the backward compatibility is really given.